pkg/utils/devlink.go

```diff
@@ -0,0 +1,96 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+)
+
+const (
+	// fwAppNameKey is a key to extract DDP name
+	fwAppNameKey = "fw.app.name"
+	pciBus       = "pci"
+)
+
+// IsDevlinkDDPSupportedByPCIDevice checks if DDP of PCI device can be acquired with devlink info command
+func IsDevlinkDDPSupportedByPCIDevice(device string) bool {
+	return IsDevlinkDDPSupportedByDevice(pciBus, device)
+}
+
+// IsDevlinkDDPSupportedByDevice checks if DDP of device can be acquired with devlink info command
+func IsDevlinkDDPSupportedByDevice(bus, device string) bool {
```

Comment:
Do we really need to pass hard-coded 'bus' argument to each function? When do you expect it's value won't be 'pci'?

Reply:
You're right. And I have, finally, fixed that.